buildscript.py

```diff
@@ -0,0 +1,34 @@
+import os
+import redash_stmo
+from subprocess import call
+from distutils.dir_util import copy_tree
+
+from pkg_resources import iter_entry_points, resource_filename
+
+
+# Make a directory for extensions and set it as an environment variable
+# to be picked up by webpack.
+EXTENSIONS_RELATIVE_PATH = "client/app/extensions"
+EXTENSIONS_DIRECTORY = os.path.join(os.getcwd(), EXTENSIONS_RELATIVE_PATH)
```

Comment:
I think this still doesn't quite work because `os.path.dirname(__file__)` returns `/app/bin` where the script lives which is then appended to `EXTENSIONS_RELATIVE_PATH` giving `/app/bin/client/app/extensions` which isn't what we want either. Perhaps in the `Dockerfile` we can do something like `ENV ROOT_DIR=/app` and use that environment variable instead in the buildscript. What do you think?

Reply:
Ah, right, this needs to use the parent dir when we move the file into `bin/`:
`os.path.join(os.path.dirname(os.path.dirname(__file__)), EXTENSIONS_RELATIVE_PATH))`